app/views/docs/configuration.phtml

```diff
@@ -22,10 +22,143 @@
 <h2><a href="#git" id="git">Configure GitHub App</a></h2>
 <p>
     Appwrite supports automatic deployments through Git integration. 
-    In order for Appwrite to access your repos, you must configure a GitHub app to enable this integration.
+    In order for Appwrite to access your repos, you must <a href="https://docs.github.com/en/apps/creating-github-apps/about-creating-github-apps/about-creating-github-apps" target="_blank" rel="noopener">create a GitHub app</a> to enable this integration.
+    The GitHub app requires the following configurations.
 </p>
 
-<p>You'll have to configure the following environment variables.</p>
+<p>
+    For automatic Git deployment to work, Appwrite needs to receive communication from GitHub, this means your Appwrite project must be accessible on the internet.
+    If you're running on <code>localhost</code>, you need to run a proxy like <b>ngrok</b>.
```

Comment:
can we add a hyperlink to the ngrok website

Reply:
Linked